pages/common/cmuwmtopbm.md

```diff
@@ -0,0 +1,9 @@
+# cmuwmtopbm
+
+> Convert a CMU window manager bitmap into a PBM image.
```

Comment:
```suggestion
> Convert a CMU window manager bitmap to a PBM image.
```
For consistency, shouldn't we prefer "convert ... to ..." over "convert ... into ..."?